Dockerfile.redhat

```diff
@@ -38,6 +38,7 @@ ARG YUM_OV_PACKAGE=intel-openvino-runtime-centos7
 ARG build_type=dbg
 ARG debug_bazel_flags=--strip=never\ --copt="-g"\ -c\ dbg
 
+RUN dnf clean all
```

Comment:
it is not needed for ubi, there were no issues observed.

Reply:
The root cause is that when there is old base image pulled then it could contain wrong cache. `yum clean all` at the end of failing command did not remove improper docker layer. While the issue was not observed for redhat it's still highly probably that the issue may occur again here.

It's better IMHO to fix it now for both cases rather than investigate similar issue later on.